control/passivity.py

```diff
@@ -0,0 +1,61 @@
+'''
+Author: Mark Yeatman  
+Date: May 15, 2022
+'''
+
+from . import statesp as ss
```

Comment:
Please import the official API names, not the internal module file name as a name which is also a API function.

```suggestion
from .statesp import StateSpace
```

(You actually don't use anything directly)